app/lib/pdf_filler/md502_pdf.rb

```diff
@@ -191,6 +191,8 @@ def account_holder_full_name(for_joint: false)
       attributes = %w[FirstName MiddleInitial LastName NameSuffix]
       account_holder_xmls = @xml_document.css('Form502 NameOnBankAccount')
       account_holder_xml = for_joint ? account_holder_xmls[1] : account_holder_xmls[0]
+      return if account_holder_xml.nil?
```

Comment:
guarding in case the `NameOnBankAccount` is for some reason `nil` when we actually do expect it so it doesn't crash